newa/__init__.py

```diff
@@ -148,7 +148,27 @@ class Erratum(Cloneable, Serializable):
     """ An eratum """
 
     release: str
-    # builds: list[...] = ...
+    builds: list[str] = []
```

Comment:
Just commented in another PR, but in general, `[]` will not work as expected, you need `factory` to spawn new list for every instance of this class: `builds: ... = field(factory=list)`.

Reply:
Hopefully fixed :-)